Sources/EdgeEventHandle.swift

```diff
@@ -14,14 +14,15 @@ import AEPServices
 import Foundation
 
 /// Details and results for a sent EdgeRequest
-struct EdgeEventHandle: Codable {
+@objc(AEPEdgeEventHandle)
+public class EdgeEventHandle: NSObject, Codable {
 
     /// Encodes the event to which this handle is attached as the index in the events array in EdgeRequest
     let eventIndex: Int?
 
     /// Payload type
-    let type: String?
+    public let type: String?
 
     /// Event payload
-    let payload: [[String: AnyCodable]]?
+    public let payload: [[String: AnyCodable]]?
```

Comment:
If we're going to make this a public class I would advise us not to expose `AnyCodable`. The `AnyCodable` type should only be used mainly for internal encoding/decoding. 

Let's update this to be more similar to the `Event` struct in how it handles `AnyCodable`
https://github.com/adobe/aepsdk-core-ios/blob/main/AEPCore/Sources/eventhub/Event.swift#L32
https://github.com/adobe/aepsdk-core-ios/blob/main/AEPCore/Sources/eventhub/Event.swift#L111
https://github.com/adobe/aepsdk-core-ios/blob/main/AEPCore/Sources/eventhub/Event.swift#L124